README.md

```diff
@@ -25,11 +25,11 @@ Reusable, opinionated, zero-conf workflows for GitHub actions
   - [Website](#Website)
 - [Customization](#customization)
   - [Secrets](#secrets)
+    - [`GH_APP_PRIVATE_KEY`](#gh_app_private_key)
     - [`FLOWZONE_TOKEN`](#flowzone_token)
     - [`GPG_PRIVATE_KEY`](#gpg_private_key)
     - [`GPG_PASSPHRASE`](#gpg_passphrase)
     - [`NPM_TOKEN`](#npm_token)
-    - [`GHCR_TOKEN`](#ghcr_token)
```

Comment:
I assume no one is using this in production anyway 👍

Reply:
Yeah, and if someone has, it's a small change to refactor their workflows.